hydra-cluster/src/Hydra/Cluster/Util.hs

```diff
@@ -47,6 +49,14 @@ keysFor actor = do
   asSigningKey :: AsType (SigningKey PaymentKey)
   asSigningKey = AsSigningKey AsPaymentKey
 
+-- | Create and save new secret key at provided path
+-- NOTE: Uses 'TextEnvelope' format
+createAndSaveSecretKey :: FilePath -> IO (SigningKey PaymentKey)
+createAndSaveSecretKey path = do
+  (_vk, sk) <- generate genKeyPair
+  writeFileBS path $ toStrict (textEnvelopeToJSON (Just "Key used to commit funds into a Head") sk)
+  pure sk
```

Comment:
Could: use `genSigningKey` as we don't need `vk`
Could: rename function to mention `SigningKey` (the actual type used)
Could: use `writeFileLBS` to avoid `toStrict`